e2e-cypress/Jenkinsfile.template

```diff
@@ -49,27 +60,32 @@ odsComponentPipeline(
 
 def stageTest(def context) {
   stage('Integration Test') {
-    // OPTIONAL: load environment variables for Azure SSO with MSALv2; please adapt variable names to your OpenShift config
+    // Define your DEV and QA base URLs in a config map in OpenShift; please adapt variable names to your OpenShift config
     // sh "oc project <project-with-configured-secrets>"
-    // cypressUser = sh(returnStdout: true, script:"oc get secret e2euser -o jsonpath='{.data.USERNAME}' | base64 -d")
-    // cypressPassword = sh(returnStdout: true, script:"oc get secret e2euser -o jsonpath='{.data.PASSWORD}' | base64 -d")
-    // azureClientId = sh(returnStdout: true, script:"oc get secret azure -o jsonpath='{.data.AZURE_CLIENT_ID}' | base64 -d")
-    // azureClientSecret = sh(returnStdout: true, script:"oc get secret azure -o jsonpath='{.data.AZURE_CLIENT_SECRET}' | base64 -d")
-    // azureTenantId = sh(returnStdout: true, script:"oc get configmaps azure -o jsonpath='{.data.AZURE_TENANT}'") // config map values are not base64 encoded
-    // authenticatorOTPSecret = sh(returnStdout: true, script:"oc get secret azure -o jsonpath='{.data.OTP_SECRET}' | base64 -d")
+    def baseUrls = [
+      : // remove this line once you have defined the config map and uncommented the next two lines, it's only here to make the example default case work
+      // dev: sh(returnStdout: true, script:"oc get configmaps cypress-config -o jsonpath='{.data.DEV_BASE_URL}'").trim(),
+      // test: sh(returnStdout: true, script:"oc get configmaps cypress-config -o jsonpath='{.data.TEST_BASE_URL}'").trim()
+    ]
+
+    def baseUrl = baseUrls.get(context.environment ?: 'dev', 'https://www.w3schools.com') // default to W3Schools for demo purposes, replace with your own default
+
+    // Example for loading environment variables for Azure SSO; please adapt variable names to your OpenShift config,
+    // making sure to precede the variable names with the environment name in lowercase (e.g., dev_username, dev_password,
+    // test_username, test_password, etc.)
+    // cypressUser = sh(returnStdout: true, script:"oc get secret e2euser -o jsonpath='{.data.${context.environment}_username}' | base64 -d")
+    // cypressPassword = sh(returnStdout: true, script:"oc get secret e2euser -o jsonpath='{.data.${context.environment}_password}' | base64 -d")
```

Comment:
@roicarrera credentials (secrets of type username and password) need to be consumed in a safe way so that they do not leak into the log stream. Please use Jenkins' ``withCredentials`` task. You can refer to other Quickstarters to see its use.